packages/mirrorful/editor/src/pages/api/longPoll.ts

```diff
@@ -0,0 +1,10 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+
+// to handle the requests
+import pool from '../../utils/connections'
+
+export default function handler(req: NextApiRequest, res: NextApiResponse) {
+  res.setHeader('Transfer-Encoding', 'chunked')
+  res.setHeader('Content-Type', 'text/html')
+  pool.push(res)
```

Comment:
I'm a bit confused, is `pool` just an empty array? Does it need to be imported? Or can we just do a res.send?

Reply:
The pool contains all the requests. What I was trying to do was when the server received the 'sigterm'  or 'sigint', it would loop over the pool and send a message back saying exiting. But since the server closes too fast, the pool just stores the request. We could potentially remove the pool but we cannot send a response back as that would end the long poll.

Reply:
Removed pool as it is not needed!